examples/single_agent.py

```diff
@@ -21,17 +21,15 @@ def act(self, obs: Observation):
             len(obs.via_data.near_via_points) < 1
             or obs.ego_vehicle_state.edge_id != obs.via_data.near_via_points[0].edge_id
         ):
-            return "keep_lane"
+            return (obs.waypoint_paths[0][0].speed_limit, 0)
```

Comment:
Why is 0?

Reply:
The `LanerWithSpeed` uses land index difference `{-1,0,1}`.

Reply:
My mistake. I was thinking about why setting the speed to 0.